Documentation/Development.md

```diff
@@ -54,6 +54,23 @@ After building, the server will be located at `.build/debug/sourcekit-lsp`, or a
 
 SourceKit-LSP is designed to build against the latest SwiftPM, so if you run into any issue make sure you have the most up-to-date dependencies by running `swift package update`.
 
+### Windows
+
+The user must provide the following dependencies for SourceKit-LSP:
+- SQLite3
+- ninja
+
+```cmd
+> swift build -Xcc -I...\sqlite-3.38.0\usr\include -Xlinker -L...\sqlite-3.38.0\usr\lib -Xcc -I%SDKROOT%\usr\include -Xcc -I%SDKROOT%\usr\include\Block
```

Comment:
IIUC the `...` need to be expanded to absolute paths based on your comment below? Maybe it’s because I’m not too familiar with Windows builds but I thought this was some way of configuring the parent directory. Something like `C:\absoulte\path\to\sqlite-3.38.0\usr\include` would have been clearer to me (not sure how best to represent the volume letter though)